lib/ohai/plugins/openstack.rb

```diff
@@ -61,7 +70,7 @@ def openstack_provider
 
   collect_data do
     # fetch data if we look like openstack
-    if openstack_hint? || openstack_dmi?
+    if openstack_hint? || openstack_dmi? || openstack_virtualization?
```

Comment:
I think this DMI check method becomes redundant as long as the virtualization plugin contains all the same manufacturer IDs. I’m on thumbs at the moment so I’ll check on this when I get to a real computer

Reply:
I've got no problem making that change in this PR, I can even move the tests to make that work. Just let me know.

Reply:
I think it's fine, since it calls the same function under the hood: https://github.com/chef/ohai/blob/master/lib/ohai/plugins/linux/virtualization.rb#L135 - and I've confirmed a linux openstack box does report `"system":"openstack"`.

Reply:
I'll clean it up post merge. There's just no point in the extra logic now that they use the same logic under the hood.

Reply:
Sure, thanks.